client/header/activity-panel/panels/inbox/test/index.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import InboxNoteCard from '../card';
+import { getUnreadNotesCount, hasValidNotes } from '../utils';
+
+jest.mock( '../action', () =>
+	jest.fn().mockImplementation( () => <button>mocked button</button> )
+);
+
+describe( 'InboxNoteCard', () => {
+	const note = {
+		id: 1,
+		name: 'wc-admin-wc-helper-connection',
+		type: 'info',
+		title: 'Connect to WooCommerce.com',
+		content: 'Connect to get important product notifications and updates.',
+		status: 'unactioned',
+		date_created: '2020-05-10T16:57:31',
+		actions: [
+			{
+				id: 1,
+				name: 'connect',
+				label: 'Connect',
+				query: '',
+				status: 'unactioned',
+				primary: false,
+				url: 'http://test.com',
+			},
+		],
+		layout: 'plain',
+		image: '',
+		date_created_gmt: '2020-05-10T16:57:31',
+		is_deleted: false,
+	};
+	const lastRead = 1589285995243;
+
+	test( 'should render a notification type banner', () => {
+		note.layout = 'banner';
+		const { container } = render(
+			<InboxNoteCard
+				key={ note.id }
+				note={ note }
+				lastRead={ lastRead }
+			/>
+		);
+		const listNoteWithBanner = container.querySelector( '.banner' );
+		expect( listNoteWithBanner ).not.toBeNull();
+	} );
+
+	test( 'should render a notification type thumbnail', () => {
+		note.layout = 'thumbnail';
+		const { container } = render(
+			<InboxNoteCard
+				key={ note.id }
+				note={ note }
+				lastRead={ lastRead }
+			/>,
+			{
+				disableLifecycleMethods: true,
```

Comment:
Curious - why was this needed for this test?

Reply:
Thanks Jeff. Yes, it's unnecessary. It was removed in the commit 8d6d9a0